web/src/hashchange.js

```diff
@@ -50,6 +51,18 @@ function maybe_hide_inbox() {
     return false;
 }
 
+function show_inbox_view() {
+    compose_actions.on_view_change();
+    maybe_hide_recent_view();
+    inbox_ui.show();
+}
+
+function show_recent_view() {
+    compose_actions.on_view_change();
+    maybe_hide_inbox();
+    recent_view_ui.show();
+}
```

Comment:
We already have a `views_util` module intended for this sort of common code to be called when switching into inbox or recent conversations; let's use that rather than the refactoring in this file.

Reply:
Thank you for the insight, that also helped simplify the code. Addressed in the new code.

Reply:
Should we also add a comment line for the new `on_view_change` function? I've omitted it atm because I felt that the logic could be inferred from the adjacent functions - `on_topic_narrow` and `on_narrow`.

Reply:
I think the new function could possibly benefit from a comment, also posted a comment about naming; otherwise lgtm.